mautic/mautic
# [UX] New default dashboard templates

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR adds 2 new dashboard templates:
- Forms and Assets
- Landing pages

using related widgets in each.
The first aims to show submissions, related downloaded assets and identified contacts.
The second aims to show page visits, all contacts created (including anonymous) and a bit of assets.






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the Dashboard, click on the page actions then Select pre-defined dashboard.
3. Check the list and test the 2 new ones.
